### DISCUSSION TITLE
[BUG] Issue on Copy-PnPList, anyone working on this?

### DISCUSSION THREAD

**Discussion by radityoardi** [pnp/powershell#3043] [Q&A]
I found an issue on Copy-PnPList cmdlet, related to copying list with a lookup field.
The code inside is actually generating an SP List script to be executed on another site, which is working great. But once a lookup field is introduced, the generated script stops working. After digging, the script generates for 2 or more different SP Lists to be created, particularly because of the presence of the lookup field.

I'm going to work on this, just asking here if anyone is working on it?

**Comment by shanprojects** [comment]
Hello, @radityoardi - Just came across your post, and I am facing exactly the same issue. 
Have you found any solution yet ? It has been a while since you posted, pls advise if you found any alternatives ?

- thank you in advance :-)

**Comment by kesterl** [comment]
I believe this bug is the same as the one reported in https://github.com/pnp/powershell/issues/3866, which was purportedly fixed by https://github.com/pnp/powershell/pull/3870. But the problematic behavior is still occurring.
